spec/features/publish/courses/new_apprenticeship.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+
+feature "selecting a teaching apprenticeship" do
+  before do
+    given_i_am_authenticated_as_an_ab_provider_user
+    when_i_visit_the_apprenticeship_page
+  end
+
+  scenario "selecting yes" do
+    when_i_select(:yes)
+    and_i_click_continue
+    then_i_am_met_with_the_full_or_part_time_page
+  end
+
+  scenario "selecting no" do
+    when_i_select(:no)
+    and_i_click_continue
+    then_i_am_met_with_the_full_or_part_time_page
+  end
+
+  scenario "invalid entries" do
+    and_i_select_nothing
+    and_i_click_continue
+    then_i_am_met_with_errors
+  end
+
+private
+
+  def given_i_am_authenticated_as_an_ab_provider_user
+    @user = create(:user)
+    @user.providers << create(:provider, :accredited_body)
+    given_i_am_authenticated(user: @user)
+  end
+
+  def when_i_visit_the_apprenticeship_page
+    new_apprenticeship_page.load(provider_code: provider.provider_code, recruitment_cycle_year: Settings.current_recruitment_cycle_year, query: outcome_params)
+  end
+
+  def when_i_select(choice)
+    new_apprenticeship_page.send(choice).click
+  end
+
+  def and_i_select_nothing; end
```

Comment:
redundant?

Reply:
I think it makes sense in the context

``` ruby
scenario "invalid entries" do
    and_i_select_nothing
    and_i_click_continue
    then_i_am_met_with_errors
  end
```

Did you mean something else?

Reply:
think we could just remove it - clearly the fact that the step is missing shows errors will be provided

Reply:
fixed in https://github.com/DFE-Digital/teacher-training-api/pull/2471/commits/e93bb0661917f03b1241f755cb91786fa403f2c6